packages/dd-trace/src/lambda/handler.js

```diff
@@ -25,7 +25,7 @@ function checkTimeout (context) {
   const remainingTimeInMillis = context.getRemainingTimeInMillis()
 
   let apmFlushDeadline = parseInt(process.env.DD_APM_FLUSH_DEADLINE_MILLISECONDS)
-  if (!apmFlushDeadline) {
+  if (!apmFlushDeadline || apmFlushDeadline < 0) {
```

Comment:
Let's keep this consistent between node and [python](https://github.com/DataDog/dd-trace-py/pull/5176/files#diff-6c80e8ef43534becbd3732c7b7631ac8a28b559325e8286df77e2e7dd7e6e9e1R66). `if apmFlushDeadline < 0`, just use `0`

Reply:
Updated the one in python to default to a `100` too, as discussed in our meeting